lib/apache_svn_url_parser.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+class ApacheSvnUrlParser < URLParser
+  private
+
+  def full_domain
+    'https://svn.apache.org/viewvc'
+  end
+
+  def tlds
+    %w(org)
+  end
+
+  def domain
+    'svn.apache'
+  end
+
+  def domain_regex
+    # match only the viewvc endpoint at the domain
+    "#{domain.split("/").first}\.(#{tlds.join('|')})\/viewvc"
+  end
+
+  def remove_domain
+    # find the matches for any github domain characters in the url string
+    # and replace only the first match incase we find a repo with something like github.com as the name
+    url.sub!(/(apache\.org\/(viewvc|repos))+?(:|\/)?/i, '')
+  end
+
+  def extractable_early?
+    false
```

Comment:
I wasn't clear on what this method is attempting to do in the workflow, but it felt easier to disable it so the URL strings go through the whole process instead of getting picked out early with a strange result.